packages/uilib/src/lib/plugins/plugin-store/+page.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
```

Comment:
Not sure if we want to include this plugin into our library archive, does it help with debugging? Otherwise we can either remove it or simply stop supporting it.

Reply:
I never used uilib for debugging, so it makes sense to get rid of it.